src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageItemControl.xaml

```diff
@@ -71,100 +64,100 @@
           </Trigger>
         </Style.Triggers>
       </Style>
+    </ResourceDictionary>
+  </UserControl.Resources>
 
-      <!-- style of this control itself -->
-      <Style x:Key="controlStyle" TargetType="{x:Type UserControl}">
-        <Setter Property="Template">
-          <Setter.Value>
-            <ControlTemplate TargetType="{x:Type UserControl}">
-              <Border
+  <Border
                 BorderThickness="0,0,0,0"
                 BorderBrush="{DynamicResource {x:Static SystemColors.ActiveBorderBrushKey}}"
                 Background="Transparent"
                 Padding="0,8,0,0">
-                <Grid
+    <Grid
                   Margin="16,0,7,0">
-                  <Grid.ColumnDefinitions>
-                    <ColumnDefinition Width="Auto" />
-                    <ColumnDefinition Width="Auto" />
-                    <ColumnDefinition Width="*" />
-                    <ColumnDefinition Width="Auto" />
-                  </Grid.ColumnDefinitions>
+      <Grid.ColumnDefinitions>
+        <ColumnDefinition Width="Auto" />
+        <ColumnDefinition Width="Auto" />
+        <ColumnDefinition Width="*" />
+        <ColumnDefinition Width="Auto" />
+      </Grid.ColumnDefinitions>
 
-                  <!-- check box -->
-                  <CheckBox
+      <!-- check box -->
+      <CheckBox
                     Grid.Column="0"
                     Margin="0,12,8,0"
                     VerticalAlignment="top"
                     Visibility="{Binding CheckBoxesEnabled, RelativeSource={RelativeSource AncestorType={x:Type nuget:InfiniteScrollList}}, Converter={StaticResource BooleanToVisibilityConverter}}"
+                    AutomationProperties.Name="{x:Static nuget:Resources.CheckBox_Selected}"
```

Comment:
I'm not this is correct.
It will announce selected even if the checkbox is not checked.

Reply:
Let me go test. We need a word though...which word would you prefer?

Reply:
My only concern is whether it saying "selected" while the checkbox is not checked could confuse the customer. 
Basically I'd expect a different thing announced depending on whether the checkbox is selected, but maybe I'm missing something.

Reply:
understood. i'll look at other examples. since it is a checkbox it will say if it is checked or not. we need to communicate that you are selected which packages will be updated by checking the box, as you know.

Reply:
we tested with narrator.. Current fix works well here. Nikolche agreed.